src/snowcli/cli/registry/commands.py

```diff
@@ -24,7 +25,10 @@
 @global_options_with_connection
 def token(**options) -> ObjectResult:
     """
-    Gets the token from environment to use for authenticating with the registry.
+    Gets the token from environment to use for authenticating with the registry. Example usage with docker:
+
+    snow snowpark registry token --format=JSON | docker login YOUR_HOST -u 0sessiontoken --password-stdin
```

Comment:
`snow snowpark registry` is not present on the main branch

Reply:
Good catch, fixed